tests/quick.rs

```diff
@@ -1362,49 +1362,29 @@ mod compress_command {
         let sample = Path::new("tests/input/quick/sample3.ref");
 
         let mut cmd = command();
-        cmd.arg("--compress")
-            .arg("-1")
-            .arg("--keep")
-            .arg("--stdout")
-            .arg("-v")
-            .arg(sample)
-            .stdout(Stdio::piped());
-
-        let output = cmd.output().unwrap();
-
-        assert!(
-            output.status.success(),
-            "{}",
-            String::from_utf8_lossy(&output.stderr)
-        );
-
-        assert_eq!(
-            String::from_utf8_lossy(&output.stderr).replace(bzip2_binary(), "bzip2"),
+        expect_success!(
```

Comment:
this helper turns the `\r\n` into just `\n`, so that the comparison with the formatted string below succeeds on windows.